src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -101,10 +169,13 @@ const SpendingByCFDAContainer = ({ onRedirectModalClick, activeTab }) => {
         if (activeTab !== 'all') {
             params.filter.award_type_codes = awardTypeGroups[activeTab];
         }
-        const request = fetchSpendingByCfda(params);
+        let request = fetchSpendingByCfda(params);
```

Comment:
Wonder if it would be better to put the logic in line 173 - 175 into the fn `fetchSpendingByCfda` or just invoke a new function that handles this?

Just an idea.